### DISCUSSION TITLE
We've had one yes, but what about second background color?

### DISCUSSION THREAD

**Discussion by Dragonoftheworld** [textadventures/quest#1255] [Creating games with Quest]
So I created a div for another background-color since I have an image that I want between body.backgroundcolor and '#backcolor'.backgroundcolor so that it overlays my image. The problem is that my custom background color also overlays the text. The solution was to add position.relative to '#gameBorder'. I don't know why it didn't have that before. Is this something that could potentially cause problems?
```
JS.eval ("$('body').prepend('<div id=\"backcolor\" style=\"position:fixed;background-image:linear-gradient(to bottom, rgba(255,255,255,0) 0%, rgba(0,102,0,1) 6%);height:100%;width:100%;\"></div>');")

JS.eval ("$('#gameBorder').css('position','relative');")
```

**Comment by KVonGit** [comment]
Hello,

I'm not much help with CSS, but I would test the game on different devices with different screen sizes to see if it looks okay. If it does, I would think it should be okay.

Someone around here who knows more about CSS will most assuredly pop in soon, though.

**Reply by KVonGit** [reply]
PS

Nice discussion title! Makes me hungry for some reason :)

**Reply by Dragonoftheworld** [reply]
Hehe thank you😁

**Reply by KVonGit** [reply]
Hello,

Is this working, as far as you can tell? (Just curious. If it is, I shall bookmark it for future reference. If unsure, I'll try to help break it ...  I mean test it. `:)`)

**Reply by Dragonoftheworld** [reply]
Hey

It is working with no problem. Though I'm not sure if relative position for gameBorder could cause problems in the future. But so far so good.
